flytekit/clis/sdk_in_container/register.py

```diff
@@ -148,14 +164,30 @@ def register(
     """
     see help
     """
+    if copy is not None and non_fast:
+        raise ValueError("--non-fast and --copy cannot be used together. Use --copy none instead.")
+
+    # Error handling for non-fast/copy conflicts
```

Comment:
Since `copy`  and `non_fast` can not be used together, there should be no conflicts?